src/main/scala/mesosphere/marathon/api/LeaderProxyFilter.scala

```diff
@@ -294,13 +294,13 @@ class JavaUrlConnectionRequestForwarder @Inject() (
             () => cloneResponseEntity(leaderConnection, response)
           )
         } finally {
-          Try(leaderConnection.getInputStream.close())
-          Try(leaderConnection.getErrorStream.close())
+          Closeables.closeQuietly(leaderConnection.getInputStream())
+          Closeables.closeQuietly(leaderConnection.getErrorStream())
         }
       }
     } finally {
-      Try(request.getInputStream.close())
-      Try(response.getOutputStream.close())
+      Closeables.closeQuietly(request.getInputStream())
+      Closeables.close(response.getOutputStream(), true)
```

Comment:
Why is the output stream not closed quietly?

Reply:
It is. The parameter is called `swallowIOException`. There is not `closeQuietly` for output streams.